src/diffusers/models/cross_attention.py

```diff
@@ -97,6 +102,31 @@ def __init__(
         processor = processor if processor is not None else CrossAttnProcessor()
         self.set_processor(processor)
 
+    def set_weights(self):
```

Comment:
this is not used anymore and should be removed before merge, just leaving it here for now to show how Conv2D weights can be converted easily to nn.Linear weights for attention